apps/web/modules/videos/ai/ai-transcribe.tsx

```diff
@@ -119,64 +71,54 @@ export const CalAiTranscribe = ({
     }, [])
   );
 
-  useDailyEvent("joined-meeting", (ev) => {
-    if (enableAutomaticTranscription) {
+  useDailyEvent("joined-meeting", () => {
+    if (enableAutomaticTranscription && !transcription?.isTranscribing) {
       daily?.startTranscription();
-      updateCustomTrayButtons({
-        recording: BUTTONS.START_RECORDING,
-        transcription: transcription?.isTranscribing
-          ? BUTTONS.STOP_TRANSCRIPTION
-          : BUTTONS.START_TRANSCRIPTION,
+    }
+    if (enableAutomaticRecordingForOrganizer && !recording?.isRecording) {
+      daily?.startRecording({
+        // 480p
+        videoBitrate: 2000,
       });
```

Comment:
Why are we handling the recording stuff in ai-transcribe? It should be a separate module or just somewhere else.

Reply:
Great point. I renamed the file and component now